OptimoveSDK/Sources/Classes/Optimobile/OptimoveInApp.swift

```diff
@@ -50,35 +53,38 @@ public class InAppInboxItem {
         return true
     }
 
-    public func isRead() -> Bool {
+    func isRead() -> Bool {
         return readAt != nil
     }
 
-    public func getImageUrl() -> URL? {
+    func getImageUrl() -> URL? {
         return getImageUrl(width: InAppInboxItem.defaultImageWidth)
     }
 
-    public func getImageUrl(width: UInt) -> URL? {
+    func getImageUrl(width: UInt) -> URL? {
         if let imagePathNotNil = imagePath {
-            return MediaHelper.getCompletePictureUrl(pictureUrl: imagePathNotNil, width: width)
+            return try? mediaHelper.getCompletePictureUrl(
+                pictureUrlString: imagePathNotNil,
+                width: width
+            )
         }
 
         return nil
     }
 }
 
-public struct InAppInboxSummary {
-    public let totalCount: Int64
-    public let unreadCount: Int64
+struct InAppInboxSummary {
+    let totalCount: Int64
+    let unreadCount: Int64
 }
 
-public typealias InboxUpdatedHandlerBlock = () -> Void
-public typealias InboxSummaryBlock = (InAppInboxSummary?) -> Void
+typealias InboxUpdatedHandlerBlock = () -> Void
+typealias InboxSummaryBlock = (InAppInboxSummary?) -> Void
 
-public enum OptimoveInApp {
+enum OptimoveInApp {
```

Comment:
Will these method still be exposed to the consumer with the new internal access modifier? [Our in app docs](https://github.com/optimove-tech/Optimove-SDK-iOS/wiki/in-app)

Reply:
It will not. It is a bug here. We have to return it back to `public`.